diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -210,6 +201,12 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    private fun ASTNode.getBreakNodeFromIf(type: IElementType) = this
+        .treeParent
+        .findChildByType(type)?.let {
+            it.findChildByType(BLOCK) ?: it
+        }?.hasChildOfType(BREAK) ?: false
```

Comment:
How does diktat allow such formatting? I would expect it to force this:
```kotlin
 private fun ASTNode.getBreakNodeFromIf(type: IElementType) = this
        .treeParent
        .findChildByType(type)
        ?.let {
            it.findChildByType(BLOCK) ?: it
        }
        ?.hasChildOfType(BREAK) 
        ?: false
```
Please create an issue from this comment, we should investigate this

Reply:
(Opened)[https://github.com/cqfn/diKTat/issues/865]